system-test/datastore.ts

```diff
@@ -1014,6 +1249,22 @@ describe('Datastore', () => {
         assert.deepStrictEqual(results, [{total: 7}]);
       });
     });
+    describe('with multiple types of filters', () => {
+      it('should run multiple types of aggregations with a list of aggregates', async () => {
+        const q = datastore.createQuery('Character');
+        const aggregate = datastore
+          .createAggregationQuery(q)
+          .addAggregations([
+            AggregateField.count(),
```

Comment:
I would add one more test here using a combination of aliases and non aliases (aka count with an alias, sum without an alias, avg with an alias all in one query)

Reply:
Done

Reply:
https://github.com/googleapis/nodejs-datastore/pull/1097/commits/bbff19ff13c0eabb19dee64536b2e493d5982cf9